static/src/javascripts/projects/facia/modules/onwards/search-tool.js

```diff
@@ -45,11 +48,49 @@ define([
             },
 
             handleClick: function (e) {
-                e.preventDefault();
+                var isInput = $(e.target).hasClass('js-search-tool-input'),
+                    isLink  = this.isLink(e.target);
+
+                if (isInput) {
+                    e.preventDefault();
+                    mediator.emit('autocomplete:toggle', true);
+                } else if (isLink) {
+                    e.preventDefault();
+                    $('.active').removeClass('active');
```

Comment:
Is this removing the `.active` class on the whole document? Maybe it should be something like `$(weatherWidget).find('.active').removeClass('active')` for better isolation.
Or you could keep a reference of the active element, so you don't have to query the DOM to get it.